java/ql/lib/semmle/code/java/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -114,9 +114,9 @@ private module CaptureInput implements VariableCapture::InputSig {
 
     CapturedVariable getVariable() { result = v }
 
-    Expr getSource() {
-      result = this.(VariableAssign).getSource() or
-      result = this.(AssignOp)
+    Node getSource() {
```

Comment:
I was about to write "I don't want to use `DataFlow::Node` in the input to variable capture", but then I noticed that this is an `additional` predicate (which apparently doesn't require the annotation since it's a member predicate). I'd like to move this predicate out of the input module to make this more clear. It can e.g. be inlined into `asClosureNode`.